src/main/scala/org/ergoplatform/network/ErgoNodeViewSynchronizer.scala

```diff
@@ -1228,7 +1227,7 @@ object ErgoNodeViewSynchronizer {
     // getLocalSyncInfo messages
     case object SendLocalSyncInfo
 
-    case class ResponseFromLocal(source: ConnectedPeer, modifierTypeId: ModifierTypeId, localObjects: Seq[(ModifierId, Array[Byte])])
+    case class ResponseFromLocal(source: ConnectedPeer, modifierTypeId: NetworkObjectTypeId, localObjects: Seq[(ModifierId, Array[Byte])])
```

Comment:
ScalaDoc missing

Reply:
Found no usages actually, removed the class